### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nehakalbande**

### CODE REVIEW

**Comment by kessmith** [comment]
I think this is a great addition

File: README.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ Feeling stuck amidst the course? Feel free to refer this repository or reach out
  &#9746; Means that the particular Chapter is under progress 👨‍💻 <br>
  &#9745; Means that the particular Chapter is completed 🎉
  
+ ## How does this scholarship program work?
+ ### Phase 1: Scholarship Foundations Course
```
Comment (Shivansh2407) [inline README.md:24]: Let's Keep these as bullet points and not the Headings.

**Review by Shivansh2407 — CHANGES_REQUESTED** [review]
LGTM @nehakalbande . Please update the PR and I'll merge it accordingly. :tada: 
Thanks.

File: README.md (hunk 1 above)
Comment (nehakalbande) [reply README.md]: Changes done.

**Review by Shivansh2407 — APPROVED** [review]
Great Work @nehakalbande :tada:

**Pull request merged**